test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -449,9 +449,14 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 		})
 
 		It("should report all nodes in a Ready state", func() {
-			nodeCount := eng.NodeCount()
-			log.Printf("Checking for %d Ready nodes\n", nodeCount)
-			ready := node.WaitOnReady(nodeCount, 10*time.Second, cfg.Timeout)
+			var expectedReadyNodes int
+			if eng.ExpandedDefinition.Properties.HasLowPriorityScaleset() {
+				expectedReadyNodes = eng.NodeCount()
```

Comment:
shouldn't it be the opposite? don't check number of nodes if HasLowPriorityScaleset is true is what we want, no?